modules/modules.nix

```
@@ -33,6 +33,7 @@ let
     (loadModule ./misc/submodule-support.nix { })
     (loadModule ./misc/version.nix { })
     (loadModule ./misc/xdg.nix { })
+    (loadModule ./numlock.nix { })
```

This is module is Linux only so this line should be
```suggestion
    (loadModule ./numlock.nix { condition = hostPlatform.isLinux; })
```